### PR TITLE
Show only 2 commentaires on liste and rework pagination

### PR DESCRIPTION
**Pull request by bguerout**

### CODE REVIEW

File: widget/src/components/CarrouselWidget.jsx [hunk 1]
```diff
@@ -9,6 +9,8 @@ import Avis from './common/Avis';
 import Button from './common/library/Button';
 import './CarrouselWidget.scss';
 
+let ITEMS_PAR_PAGE = 1;
```
Comment (loicguillois) [inline widget/src/components/CarrouselWidget.jsx]: const ?

File: widget/src/components/ListeWidget.jsx [hunk 2]
```diff
@@ -7,6 +7,9 @@ import Propulsed from './common/Propulsed';
 import Header from './common/Header';
 import Avis from './common/Avis';
 import './ListeWidget.scss';
+import Button from './common/library/Button';
+
+let ITEMS_PAR_PAGE = 2;
```
Comment (loicguillois) [inline widget/src/components/ListeWidget.jsx]: const ?

File: widget/src/components/CarrouselWidget.jsx (hunk 1 above)
Comment (bguerout) [reply widget/src/components/CarrouselWidget.jsx]: Done

File: widget/src/components/ListeWidget.jsx (hunk 2 above)
Comment (bguerout) [reply widget/src/components/ListeWidget.jsx]: Done

**Pull request merged**